src/Components/FriendsPage/FriendsPage.js

```diff
@@ -1,41 +1,75 @@
-import React, { useState } from 'react'
-import './FriendsPage.css'
+import React, { useState, useEffect } from 'react';
+import './FriendsPage.css';
+import { getFriends } from '../../ApiCalls';
 
 function FriendsPage() {
+  const [search, setSearch] = useState('');
+  const [friends, setFriends] = useState([]);
+  const [filteredFriends, setFilteredFriends] = useState([]);
+  const [newFriendName, setNewFriendName] = useState('');
 
-  const friends = ['Trevor', 'Gus', 'Joe', 'Jill']
-  const [search, setSearch] = useState('Search Friends');
-  const [filteredFriends, setFilteredFriends] = useState(['Trevor', 'Gus', 'Joe', 'Jill']);
+  useEffect(() => {
+    getFriends(1)
+      .then(data => {
+        setFriends(data.data.friends);
+        setFilteredFriends(data.data.friends);
+      })
+      .catch(err => console.log(`There has been an error: ${err}`))
+  }, []);
 
-  const displayFriends = filteredFriends.map(friend => {
-    return <div className='long-tile'>{friend}</div>
-  })
+  useEffect(() => {
+    filterFriends(search);
+  });
 
-  const filterSearch = value => {
+  const displayFriends = filteredFriends.map(friend => (
+    <div className='long-tile' key={friend.user_id}>
+      {friend.user_name}
+    </div>
+  ));
+
+  const filterFriends = value => {
     const filteredFriends = friends.filter(friend => {
-      return friend.includes(value);
+      return friend.user_name.toLowerCase().includes(value.toLowerCase());
     });
-    setFilteredFriends(filteredFriends)
-    filteredFriends.map(friend => {
-      return <div className='long-tile'>{friend}</div>
-    })
+    setFilteredFriends(filteredFriends);
   }
 
+  const handleSearch = event => {
+    setSearch(event.target.value);
+  };
+
+  const handleAddName = () => {
+    if (newFriendName !== '') {
+      const newFriend = { user_id: friends.length + 1, user_name: newFriendName };
+      setFriends([...friends, newFriend]);
+      setFilteredFriends([...filteredFriends, newFriend]);
+      setNewFriendName('');
```

Comment:
I think this is the piece that you DM'd about on Slack, but just want to confirm that this piece will just be a request to an end point, then with the response, we'll update `friends` in Redux rootslice state